### DISCUSSION TITLE
PRIVATE NLB TARGET IP OVER PEERED VPC CONNECTION

### DISCUSSION THREAD

**Discussion by budiprihantono** [awsugid/forum#35] [Q&A]
Halo teman2 izin bertanya, saya ada case dmn saya mempunyai 2 vpc dan terhubung oleh peering,
VPC A CIDR 10.54.0.0/20
VPC B CIDR 10.198.13.0/24

ada kebutuhan dimana saya provisioning nlb private di vpc B (ip private nlb 10.198.13.214) dengan listener TCP port 22 (disable preserve ip), untuk forward traffic ke ip 10.54.4.49 port 25322  ke instance yang ada di vpc A.

hal yg sudah dilakukan :
- security group nlb dan di target instance sudah di tambahkan
- routing dan peering sudah berhasil, saya bisa direct telnet dr salah satu instance yang ada di vpc b ke 10.54.4.49 port 22,,tetapi tidak berhasil jika telnet menggunakan 10.198.13.214 port 25322 (ip private nlb di vpc b)

issue :
- tidak berhasil jika telnet ke private nlb menggunakan 10.198.13.214 port 25322
- unhealthy target group ip 10.54.4.49 port 25322 zone all Preserve client IP disable

mungkin dr tmn2 ada masukan? terima kasih

**Comment by saiqulhaq** [comment]
coba enable Flow Logs biar tau detail masalahnya https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html
